jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/util/KubernetesResourceUtilTest.java

```diff
@@ -298,7 +300,7 @@ void mergeResources_whenDeploymentProvidedAndFirstDeploymentWithEmptySpec_thenSh
           .extracting(DeploymentSpec::getTemplate)
           .extracting(PodTemplateSpec::getSpec)
           .extracting(PodSpec::getContainers)
-          .asList()
+          .asInstanceOf(InstanceOfAssertFactories.list(Type.class))
```

Comment:
`list(..)` argument should be object that's composing previous list used in `extracting` i.e `getContainers`